Cargo.toml

```diff
@@ -1,13 +1,14 @@
 [package]
 name = "syndication"
-version = "0.1.1"
-authors = ["Tom Shen <tom@shen.io>"]
+version = "0.1.2"
```

Comment:
You should rebase your branch, since the library is already on version "0.2.0." This change should bump the version to "0.3.0," since it breaks the original API.